<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QToy2lyr</class>
 <widget class="QWidget" name="QToy2lyr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QToy2lyr</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>30</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLineEdit" name="perlPathEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="perlChooseButton">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPlainTextEdit" name="OriginalPlainTextEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>551</width>
     <height>181</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="copyFileEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>551</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>57</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Original</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>57</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Copie</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fichier  sous titre Ã  charger</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Perl exec</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="lyrPlainTextEdit">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>100</y>
     <width>451</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="frmFileEdit">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>330</y>
     <width>451</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fichier Lyr</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>310</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fichier FRM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exectoy2lyrButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>520</y>
     <width>281</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Executer toy2lyr</string>
   </property>
  </widget>
  <widget class="QPushButton" name="makeSubButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>520</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Generer sous titre</string>
   </property>
  </widget>
  <widget class="QPushButton" name="launchButton">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>520</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Lancer le fichier</string>
   </property>
  </widget>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>1071</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Info Label</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLineEdit" name="videoPathEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="videoChooseButton">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Video File</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="filePathEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="fileChooseButton">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>520</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Enregister les fichiers</string>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>OriginalPlainTextEdit</zorder>
  <zorder>copyFileEdit</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>lyrPlainTextEdit</zorder>
  <zorder>frmFileEdit</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>exectoy2lyrButton</zorder>
  <zorder>makeSubButton</zorder>
  <zorder>launchButton</zorder>
  <zorder>infoLabel</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>label_7</zorder>
  <zorder>saveButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
